backend/src/utils/jwt.js

```diff
@@ -0,0 +1,17 @@
+const jwt = require('jsonwebtoken');
+
+const createJwt = async user => {
+  return jwt.sign(
+    user,
+    process.env.JWT_SECURITY_KEY || ""
```

Comment:
Check that this var is not empty as that the top of this module to prevent the server from starting if it is empty